core/cron/crontabs.include.php

```diff
@@ -49,7 +52,27 @@
 include_once(SYS_PATH.'/core/cron/gym.cron.php');
 include_once(SYS_PATH.'/core/cron/pokemon.cron.php');
 include_once(SYS_PATH.'/core/cron/pokestop.cron.php');
-include_once(SYS_PATH.'/core/cron/nests.cron.php');
 if ($config->system->captcha_support) {
 	include_once(SYS_PATH.'/core/cron/captcha.cron.php');
 }
+
+// The following files are updated every 24h only because the queries are quite expensive
+// and they don't need a fast update interval
+$update_delay = 86400;
+$pokedex_rarity_file = SYS_PATH.'/core/json/pokedex.rarity.json';
+$nests_file = SYS_PATH.'/core/json/nests.stats.json';
+
+// Do not update both files at the same time to lower cpu load
```

Comment:
Ah sorry guys I have to clarify sth.
- Because of touch there will be only ONE running cron for rarity (or nests), because touch sets the mtime of the file to now.
- but it doesn't prevent running rarity and nests cron at the same time. I could improve this with some kind of lock file. I'll take a look.